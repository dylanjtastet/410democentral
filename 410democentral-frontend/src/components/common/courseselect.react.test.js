import React from 'react';
import renderer from 'react-test-renderer';
import Courseselect from './courseselect.jsx';

test('Component reflects group prop', () => {
    let group1 = {_id: "comp210"}
    const component1 = renderer.create(
        <Courseselect group={group1}></Courseselect>
    );

    let tree1 = component1.toJSON();

    let group2 = {_id: "comp550"}
    const component2 = renderer.create(
        <Courseselect group={group2}></Courseselect>
    );
    let tree2 = component2.toJSON();

    let group3 = {_id: "comp690"}
    const component3 = renderer.create(
        <Courseselect group={group3}></Courseselect>
    );
    let tree3 = component3.toJSON();

    expect(tree1).toMatchSnapshot();
    expect(tree2).toMatchSnapshot();
    expect(tree3).toMatchSnapshot();
})