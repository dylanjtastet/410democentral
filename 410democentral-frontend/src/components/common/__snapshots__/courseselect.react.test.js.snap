// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component reflects group prop 1`] = `
<a
  className="dropdown-item"
  href="#coursebutton"
  onClick={[Function]}
>
  comp210
</a>
`;

exports[`Component reflects group prop 2`] = `
<a
  className="dropdown-item"
  href="#coursebutton"
  onClick={[Function]}
>
  comp550
</a>
`;

exports[`Component reflects group prop 3`] = `
<a
  className="dropdown-item"
  href="#coursebutton"
  onClick={[Function]}
>
  comp690
</a>
`;
